README.md

```diff
@@ -1 +1,12 @@
-# L_Combine
\ No newline at end of file
+# L_Combine
+## Fronted
+
+Aby uruchomić aplikację kliencką należy:
+ 
+1. Mieć zainstalowanego npm'a. Można go zainstalować razem z node.js z tej strony: https://nodejs.org/en/
+2. Mieć zainstalowanego angulara. Można to zrobić przez wpisanie w terminalu: 
+npm install -g angular-cli
```

Comment:
to chyba jest npm install -g @angular/cli

Reply:
poprawione